docs/gaia/networks.md

```diff
@@ -1,9 +1,53 @@
 # Networks
 
-There are a variety of ways to setup either local or remote networks with automation, detailed below.
-All the required files are found in the [networks directory](https://github.com/cosmos/cosmos-sdk/tree/develop/networks) and additionally the `local` or `remote` sub-directories.
+This document describes 3 ways to setup a network of `gaiad` nodes, each with an increasing level of complexity:
```

Comment:
`each with an increasing level of complexity:` I'm not sure we need this? Maybe we can just say "serving various purposes or setups"?

Reply:
good call.